dbt_project.yml

```diff
@@ -542,6 +542,8 @@ models:
         +schema: addresses_events_optimism
       polygon:
         +schema: addresses_events_polygon
+      celo:
+        +schema: addresses_events_celo
```

Comment:
small nit:
This can be removed given you've added `schema = addresses_events_celo` in the model config (in `models/addresses_events/celo/addresses_events_celo_first_funded_by.sql`), which is the preferred way to set the schema. :+1: 
Note however that `models/addresses_events/celo/addresses_events_celo_first_funded_by_legacy.sql` is missing the schema in the config.

Reply:
good call out, a step towards cleaning the project out and making schema more obvious in the model itself. we can look to merge when you address this minor change @henrystats

Reply:
yes thank you ser @0xRobin! minor change addressed ser @jeff-dude